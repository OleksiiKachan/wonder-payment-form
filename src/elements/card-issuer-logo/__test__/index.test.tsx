import React from 'react';
import { shallow } from 'enzyme';
import CardIssuerLogo from '../';

describe('CardIssuerLogo', () => {
  it('Visa', () => {
    const component = shallow(<CardIssuerLogo issuer="visa" />);

    expect(component).toMatchSnapshot();
  });
  it('Mastercard', () => {
    const component = shallow(<CardIssuerLogo issuer="mastercard" />);

    expect(component).toMatchSnapshot();
  });
  it('Amex', () => {
    const component = shallow(<CardIssuerLogo issuer="amex" />);

    expect(component).toMatchSnapshot();
  });
  it('Discover', () => {
    const component = shallow(<CardIssuerLogo issuer="discover" />);

    expect(component).toMatchSnapshot();
  });
});
