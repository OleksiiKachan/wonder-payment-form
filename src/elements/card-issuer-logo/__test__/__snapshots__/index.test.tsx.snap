// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardIssuerLogo Amex 1`] = `
<img
  alt="amex"
  src="amex.svg"
/>
`;

exports[`CardIssuerLogo Discover 1`] = `
<img
  alt="discover"
  src="discover.svg"
/>
`;

exports[`CardIssuerLogo Mastercard 1`] = `
<img
  alt="mastercard"
  src="mastercard.svg"
/>
`;

exports[`CardIssuerLogo Visa 1`] = `
<img
  alt="visa"
  src="visa.svg"
/>
`;
